### PR TITLE
Business : Implement MongoDB connection + Series Crud

### PR DESCRIPTION
**Pull request by ptitdam2001**
- Update Business to implement MongoDB with one Collection Series

For the moment, the structure of Series is : 
{

  "SeriesInstanceUID": "my-id"
}

### CODE REVIEW

File: business/MetadataDatabase/Models/SeriesContext.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+namespace MetadataDatabase.Models
+{
+    using MongoDB.Driver;
+    using System;
+
+    public class SeriesContext: IModelContext<Series>
+    {
+        private readonly IMongoDatabase _db;
```
Comment (cquennouelle) [inline business/MetadataDatabase/Models/SeriesContext.cs:8]: Weird indentation, no ?

**Review by cquennouelle — APPROVED** [review]

File: client/web/src/components/series/list/SeriesList.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const SeriesList: React.FC<SeriesListProps> = () => {
 		<>
 			<div>SeriesList</div>
 			<div>{
-				list.map(({ seriesInstanceUID }) => <div key={seriesInstanceUID}>{seriesInstanceUID}</div>)
+				list.map(({ uid }) => <div key={uid}>{uid}</div>)
```
Comment (bat3) [inline client/web/src/components/series/list/SeriesList.tsx]: Let seriesInstanceUID, it is more clear than uid.

File: business/MetadataDatabase/Models/SeriesContext.cs (hunk 1 above)
Comment (ptitdam2001) [reply business/MetadataDatabase/Models/SeriesContext.cs:8]: euh no SeriesContext extends of interface IModelContext with usage of types Series

File: client/web/src/components/series/list/SeriesList.tsx (hunk 2 above)
Comment (ptitdam2001) [reply client/web/src/components/series/list/SeriesList.tsx]: ok but it is an example for the moment

**Review by bat3 — APPROVED** [review]

**Pull request merged**